astropy/utils/data.py

```diff
@@ -339,6 +330,10 @@ def get_readable_fileobj(
     if isinstance(name_or_obj, os.PathLike):
         name_or_obj = os.fspath(name_or_obj)
 
+    # FIXME: this should be fixed
```

Comment:
Can you please elaborate on what exactly should be fixed? Is there a follow-up issue?